apps/tenant-management-webapp/src/app/pages/admin/services/configuration/editor/ConfigurationData.tsx

```diff
@@ -16,23 +14,17 @@ import { useDebounce } from '@lib/useDebounce';
 import { CustomLoader } from '@components/CustomLoader';
 import { ConfigDefinition } from '@store/configuration/model';
 import { getConfigurationRevisions } from '@store/configuration/action';
+import { replaceConfigurationDataAction } from '@store/configuration/action';
 
-interface TemplateEditorProps {
-  //eslint-disable-next-line
-  setDataError?: (error: string) => void;
-  dataError?: string;
-  configurationData: string;
-  setConfigurationData: (data: string) => void;
-}
-
-export const ConfigurationData = ({
-  dataError,
-  setDataError,
-  configurationData,
-  setConfigurationData,
-}: TemplateEditorProps): JSX.Element => {
+export const ConfigurationData = (): JSX.Element => {
   const dispatch = useDispatch();
   const { id } = useParams<{ id: string }>();
+  const [namespace, name] = id.split(':') || [];
```

Comment:
Hi @jonathanweyermann  is the id the ADSP id? I am thinking of whether we can use the existing lib to determine the namespace and name.

Reply:
its just a string with namespace:name

not sure if we have a library for that